res/__tc/gcc/3_test_mpi.sh

```
@@ -0,0 +1,23 @@
+set -ex
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)